check_prometheus_metric.sh

```diff
@@ -50,6 +50,8 @@ function usage {
     -m METHOD        Comparison method, one of gt, ge, lt, le, eq, ne.
                      (Defaults to ge unless otherwise specified.)
     -C CURL_OPTS     Additional flags to pass to curl.
+                     Can be passed multiple times. Options and option values must be passed separately.
+                     e.g. -C -H -C "Authorization: Bearer $TOKEN" -C --conect-timetout -C 10
```

Comment:
This is encouraging passing secrets on the command line, which we shouldn't be doing.

Reply:
True, but any other curl options that require quoted strings will have the same problem.

Reply:
I'm suggesting that a secret not be the example.

Reply:
Yes, we should definitely change that, once we know if/how to solve this problem. (Seriously, I guess rewriting this plugin in Python would have taken less time than what we have collectively thought about Bash escaping by now…)